containers/daap-presigned-url/src/var/task/presigned_url.py

```diff
@@ -103,16 +99,15 @@ def handler(event, context):
             ),
         }
 
-    if any(data_product_registration):
```

Comment:
We can skip this extra check as we have already returned if there is no registration.